packages/vsce/__tests__/__e2e__/02_AddWiremockProfileAndListPlexesInCics.test.ts

```diff
@@ -110,18 +100,18 @@ describe("Test Suite For Adding Wiremock Profile And Listing The CICSplexes", ()
       cicsTree.takeScreenshot();
     });
 
-    it("Should Display Wiremock Profile Under CICS Section And Check For The Title Of Profile", async () => {
+    it("Should Display Wiremock Profile Under CICS Section And Title Check For The Wiremock Profile", async () => {
```

Comment:
Because these `it` functions build on each other within a "describe", they should end up writing a story:

describe("Write down our overall aim for this section")
it("Step 1 is doing this")
it("Step 2 is doing that")

So... your describe is good 👍 
This it is the first part of the story ... how about:
`it("Make sure Wiremock profile is available")`

Then your second part can be
`it("Check two plexes are available")`

and because these `it`s are part of the same overall describe, we will be able to read them and see a picture building up as you test.

Does that make sense? It might be better to talk this over on teams.